scripts/build-all.ts

```diff
@@ -21,6 +21,7 @@ const prioritisedPackages = [
 	brand,
 	`${coreComponents}/user-feedback`,
 	`${coreComponents}/label`,
+	`${coreComponents}/button`,
```

Comment:
Button is now being used by Editorial system components, so we'll need to ensure the build for Button happens before the Build for Editorial components